ENIGMAsystem/SHELL/Platforms/xlib/XLIBwindow.cpp

```diff
@@ -43,6 +44,11 @@
 
 Cursor NoCursor, DefCursor;
 
+static int window_get_min_width = -1;
+static int window_get_max_width = -1;
+static int window_get_min_height = -1;
+static int window_get_max_height = -1;
```

Comment:
Maybe we want to move these to General so that the game has them for all platfoms. We're going to want to add this to Win32 anyway with WM_GETMINMAXINFO message. I think we'll need the state there anyway, not sure.

Reply:
I can't put it in General becuase it produces the errors in my comment farther down this page.

Reply:
You got this one, this is resolved.